### DISCUSSION TITLE
未來可能人手一台…什麼都沒有！？

### DISCUSSION THREAD

**Discussion by BeaGle0928** [1101-GS4719-C/Python-Community-GS4719-C#1743] [General]
新創公司Humane發表了人工智慧硬體「AI Pin」，AI Pin是一款以大語言模型為基底的AI裝置，重量不到100克，沒有螢幕，可直接別胸上，以投影顯示結果，還可用語音與手勢與它互動。
我們都曾看過科幻電影裡那些看起來非常酷炫且令人嚮往的高科技裝置，而其中又以鋼鐵人電影裡的助理J.A.R.V.I.S.最令我印象深刻。順帶一提它的全名是Just A Rather Very Intelligent System，蠻有趣的。而AI Pin也有著相似的功能，人們可以透過聲音、手勢或手部接觸的方式來操作Ai Pin；對比我們手機內的Siri，Pin的對話顯得更為自然，不用特別喚醒裝置，也不用非得用下命令的方式才能互動，還有另一個特色就是它非常的輕薄、方便攜帶。我認為AI Pin正在嘗試定義一個全新的設備試圖改變現今的科技環境，然而對於已經非常熟悉智慧型手機的人們，這樣新穎的產品的接受度如何，仍有待討論。
具備以上種種特點，AI Pin有沒有可能取代手機成為更加普及的時代工具，抑或是成為了科技發展中的一粒細砂，隨著時間推移逐漸被掩埋消失呢？

**Comment by Ryanlolry** [comment]
我認為會是未來的趨勢但感覺在五到十年內感覺還是無法實現，畢竟手機的研發與流程蠻成熟的，新型的穿戴裝置必須有突破性的進展，或與手機具有相同功能，且更便利或更便宜才會被考慮，而現在看到的這個看似並不具備以上的形容。

**Comment by Chengbosco** [comment]
目前還很難取代吧，因為手機已經建立了龐大的應用程式和生態系統，人們習慣於使用手機進行各種活動，包括社交媒體、購物、工作等。要讓AI Pin成功取代手機，規模度還做不到。而且人們對於新科技的接受度是一個重要的考量因素。即使AI Pin提供更自然的互動方式，人們可能仍需要一些時間來適應並轉變他們的使用習慣。而且功能似乎還不夠健全。手機已經擁有豐富的功能，從攝影、視訊通話到各種應用程式。總的來說，AI Pin需要更加滿足人們對於多樣性功能的需求，才有可能成為一個全面替代手機的裝置。

**Comment by WeiYen1103** [comment]
前陣子有看到這則新聞，這真的是一大創新，一個跟apple watch大小一樣，但更輕，功能更多的AI Pin，我覺得雖然功能不完整，但認為他們的研究方向是對的!利用投影的方式，加上手勢就能控制打電話，播歌等等，但好貴買不起，不然我超想要有一台試試看會是甚麼樣子的使用體驗。

**Comment by mrweixd** [comment]
hologram用想的就帥得不得了，我們對未來發展的速度總是一無所知，如同萊納兄弟的飛行一樣，只能說我們可以期待他的發生。We will see.

**Comment by davidteng00000** [comment]
AI Pin的輕薄、自然的互動方式帶來全新體驗。然而，要取代手機需突破更多障礙。現有裝置成熟，新裝置若無同等功能或更便利，難獲廣泛接受。

**Comment by eric202501** [comment]
不論社會或個人已經對手機已經建立了深厚的依賴，而AI Pin需要獲得民心並具有獨樹一幟的優勢。它的優勢在於輕薄便攜、自然互動，但這也需要社會普及和接受的時間。未來的發展將取決於它是否能夠有效解決人們的需求，並且是否能夠適應變化中的市場。

**Comment by chuanmin0317** [comment]
目前不太能想像那一天的到來，畢竟目前來說，使用手機時，滑手機螢幕的回饋感不太能被手勢取代，不過這種高科技設備的發明讓電影中的情節在現實生活中實現，也確實令人振奮

**Comment by TobywithDino** [comment]
這讓我想到之前好像有聽說有人在開發耳部裝置，把晶片放入耳骨或腦附近，就可以像是無線耳機一樣，接受訊號然後聽因為或是跟別人講電話，雖然感覺起來很方便，但如果有壞蛋想使壞，他可以駭入你的裝置，然後播放超大分貝德的噪音，感覺會生不如死
